manala.haproxy/lookup_plugins/manala_haproxy_configs.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
You should also remove `manala.haproxy/lookup_plugins/__pycache__/manala_haproxy_configs.cpython-37.pyc` and `manala.haproxy/lookup_plugins/manala_haproxy_configs.pyc`:)